pkg/app/web/package.json

```diff
@@ -51,7 +51,7 @@
     "file-loader": "^6.2.0",
     "fork-ts-checker-webpack-plugin": "^6.2.10",
     "html-webpack-plugin": "^5.3.1",
-    "jest": "^27.0.5",
+    "jest": "27.4",
     "jest-environment-jsdom": "^27.0.5",
```

Comment:
please update this `jest-environment-jsdom` as well since it's the cause of this issue 🙏